flows/contact.go

```diff
@@ -262,7 +264,7 @@ func (c *Contact) ResolveQueryKey(key string) []interface{} {
 			switch typed := fieldValue.(type) {
 			case nil:
 				return nil
-			case *Location:
```

Comment:
this where its useful to have `LocationPath`  be a type rather than just a string